netbox/netbox/views.py

```diff
@@ -100,7 +96,7 @@
     }),
     ('virtualchassis', {
         'queryset': VirtualChassis.objects.prefetch_related('master').annotate(
-            member_count=Count('members')
+            member_count=Count('members', distinct=True)
```

Comment:
This is probably okay, but per my comment on #4903, we should ensure that the fix is applied consistently everywhere VirtualChassisFilterSet is used.